### ISSUE TITLE
Admin Columns for Taxonomies

### ISSUE THREAD

**Issue by stompweb** [johnbillion/extended-taxos#16]
With the term meta coming in 4.4. could something similar to the CPT admin columns be added to this plugin?

```php
admin_cols' => array(
        'featured_image' => array(
            'title'          => 'Illustration',
            'featured_image' => 'thumbnail'
        ),
        'published' => array(
            'title'       => 'Published',
            'meta_key'    => 'published_date',
            'date_format' => 'd/m/Y'
        ),
        'genre' => array(
            'taxonomy' => 'genre'
        )
    )
```

So it would take effect here:

![alt text](https://d7x7yelf3gv9s.cloudfront.net/content/uploads/2015/06/Pixie-Article-Audiences.png "Admin Columns Taxonomies")

**Comment by johnbillion** [comment]
Coincidentally I just wontfixed #10. I think term meta is a good reason to add custom column functionality though.

**Comment by stompweb** [comment]
Yeah I think having the description and slug, for example, might not be the most important data about the term now we have the term meta.

**Comment by johnbillion** [comment]
Fixed in version 2.0.0. There's now an `admin_cols` argument similar to Extended CPTs. See inline docs for details.
